bodhi/models/models.py

```diff
@@ -1044,6 +1044,7 @@ def set_request(self, db, action, username):
         topic = u'update.request.%s' % action
         if action is UpdateRequest.unpush:
             self.unpush()
+            self.request = None
```

Comment:
Would it make more sense to set the request to None *inside* the definition of the ``unpush`` method?  That way, we could be sure the request is correct whenever that method is called.